src/Wallabag/CoreBundle/Controller/RssController.php

```diff
@@ -18,40 +17,52 @@ class RssController extends Controller
     /**
      * Shows unread entries for current user.
      *
-     * @Route("/{username}/{token}/unread.xml", name="unread_rss", defaults={"_format"="xml"})
+     * @Route("/feed/{username}/{token}/unread/{page}", name="unread_rss", defaults={"page": 1})
+     * @Route("/{username}/{token}/unread.xml", defaults={"page": 1})
```

Comment:
Don't remove the `_format`, it send the proper content type.
Otherwise, send the proper content type your self (in `showEntries`)